pkg/extensions/search/cve/cve_test.go

```diff
@@ -457,90 +446,6 @@ func TestDownloadDB(t *testing.T) {
 	})
 }
 
-func TestImageFormat(t *testing.T) {
```

Comment:
Did this test case get removed? why so?

Reply:
https://github.com/anuvu/zot/pull/172/files#diff-b86e11fa5a3102b336caebec3b30a9d35e26af554dd8658f124dba2404b7d24aR156